lib/ReactViews/Feedback/FeedbackForm.jsx

```diff
@@ -87,6 +95,13 @@ const FeedbackForm = React.createClass({
                       <input type="text" name="email" className={Styles.field} value={this.state.email} onChange={this.handleChange}/>
                       <label>Comment or question</label>
                       <textarea className={Styles.field} name="comment" value={this.state.comment} onChange={this.handleChange}/>
+                      <div className={classNames(Styles.shareUrl)}>
```

Comment:
Since you have only one style, you don't need `classNames`.  See `Styles.action` below.